modules/client/test/integration/constants.ts

```diff
@@ -46,7 +46,7 @@ const grapqhlEndpoints = {
   working: [
     {
       url:
-        "https://api.thegraph.com/subgraphs/name/aragon/aragon-zaragoza-goerli",
+        "https://api.thegraph.com/subgraphs/name/josemarinas/josemarinas-core-goerli",
```

Comment:
Let's either `env` parameterize this or keep the standard URL

Reply:
Yes, once the subgraph is deployed I'll make a commit updating that